src/pages/Compendia/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import classnames from 'classnames';
+import { Link, withRouter, Redirect } from 'react-router-dom';
+import './Compendia.scss';
+import PointsBackground from './PointsBackground';
+import NormalizedCompendia from './NormalizedCompendia';
+import RNASeqSampleCompendia from './RNASeqSampleCompendia';
+import EmailSection from '../Main/EmailSection';
+import { useTheme } from '../../common/ThemeContext';
+
+const Compendia = function Compendia({ location, replace }, ...props) {
```

Comment:
What is `...props` here for? Also, you could probably remove the variable declaration

Reply:
removed